employees/forms.py

```diff
@@ -96,15 +96,9 @@ def __init__(self, *args: Any, **kwargs: Any):
     def _filter_task_activities_per_project(self) -> None:
         if "project" in self.data:
             with suppress(ValueError, TypeError):
-                project_id = int(self.data.get("project"))
-                self.fields["task_activities"].queryset = TaskActivityType.objects.filter(projects=project_id).order_by(
-                    "name"
-                )
+                 self.fields["task_activities"].queryset = TaskActivityType.objects.all()
         elif self.instance.pk:
-            self.fields["task_activities"].queryset = (
-                self.instance.project.project_activities.order_by("name")
-                | TaskActivityType.objects.filter(pk=self.instance.task_activities.pk)
-            ).distinct()
+            self.fields["task_activities"].queryset = (TaskActivityType.objects.all()).distinct()
```

Comment:
`distinct` is not needed here also because you don't need to check whether in queryset are any duplicates

Reply:
changed